sites/partners/src/components/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -82,6 +82,9 @@ const FormUnits = ({
   let nextId
   if (enableUnitGroups) {
     nextId = unitGroups && unitGroups.length > 0 ? unitGroups[unitGroups.length - 1]?.tempId + 1 : 1
+    if (listingAvailability !== "openWaitlist") {
+      setValue("listingAvailabilityQuestion", "openWaitlist")
+    }
```

Comment:
I'm not sure about that. In Detroit we don't have it, but in core that value set to `openWaitlist` seem to solve saving issue. But now it's missing `How is the application review order determined?` section 🤔

Reply:
@ludtkemorgan We don't show the `How is the application review order determined?` question in Detroit, so that makes sense. But, if `listingAvailabilityQuestion` is set to `openWaitlist`, it looks like we can't save a lottery event. This didn't matter before in Detroit, but @ludtkemorgan confirming that Detroit anticipates having both unit groups and lotteries?

Reply:
I think Lotteries are a thing that Detroit will want. So we should anticipate handling that case.